tests/Application/src/Tests/Controller/SubscriptionJsonApiTest.php

```diff
@@ -36,9 +36,19 @@ public function it_allows_showing_a_subscription(): void
         ;
 
         $this->client->request('GET', '/ajax/subscriptions/' . $subscription->getId());
-        $response = $this->client->getResponse();
 
-        $this->assertResponse($response, 'subscriptions/show_response', Response::HTTP_OK);
+        $this->assertResponseIsSuccessful();
+        $this->assertResponseStatusCodeSame(Response::HTTP_OK);
+        $this->assertResponseHeaderSame('content-type', 'application/json; charset=utf-8');
```

Comment:
Do we need a charset check? If so, then in all other files as well?
```suggestion
        $this->assertResponseHeaderSame('content-type', 'application/json');
```

Reply:
Yes, normally we check that in a lot of test files.